src/components/FeaturedEventsFilter.tsx

```diff
@@ -0,0 +1,464 @@
+								})
+							}}>
+							<CancelIcon />
+						</span>
+					</FilterTiles>
+				)}
+				{filterList.title.length > 0 && (
+					<FilterTiles>
+						<span className="selected-text">Title</span>
+						<span
+							className="icon"
+							onClick={() => {
+								deleteParams('title')
+								setFilterList(prev => {
+									return { ...prev, title: [] };
+								})
+							}}>
+							<CancelIcon />
+						</span>
+					</FilterTiles>
+				)}
+			</FilterTilesWrapper>
+		</FEWrapper>
+	);
+};
+
+export default FeaturedEventsFilter;
+
+const FEWrapper = styled.div`
+	margin-bottom: 1rem;
+
+	@media ${({ theme }) => theme.breakpoints.above.md} {
+		display: flex;
+		flex-direction: column;
+		align-items: center;
+		margin-bottom: 2rem;
+	}
+`;
+
+const FilterTilesWrapper = styled.div`
+	display: flex;
+	gap: 1rem;
+	margin-top: 1rem;
+`;
+
+const FilterTiles = styled.span`
+	display: inline-flex;
+	align-items: center;
+	background-color: #ffe8da;
+	border-radius: 9px;
+	padding: 2px 8px;
+
+	& .icon {
+		cursor: pointer;
+	}
+	& .selected-text {
+		font-size: 1.2rem;
+	}
+`;
+
+const FilterWrapper = styled.div`
+	& .filter-btn {
+		font-size: 1rem;
+		padding: 0.6rem 1.2rem;
+		margin-top: 0.7rem;
+	}
+
+	@media ${({ theme }) => theme.breakpoints.above.md} {
+		margin: 1rem 0;
+		display: flex;
+
+		& .filter-btn {
+			font-size: 1rem;
+			padding: 0.8rem 2rem;
+			margin-top: 0rem;
+		}
+	}
+`;
+const FilterInputs = styled.div`
+	background-color: white;
+	display: flex;
+	flex-direction: column;
+
+	@media ${({ theme }) => theme.breakpoints.above.md} {
+		gap: 1rem;
+		flex-direction: row;
+	}
+`;
+
+const CheckBoxGroupWrapper = styled.div`
+	position: relative;
+	display: inline-block;
+	width: 200px;
+	cursor: pointer;
+
+	&:hover .checkbox-title {
+		color: var(--oex-orange);
+	}
+
+	&:hover .options {
+		display: block;
+	}
+
+	& .options {
+		display: none;
+		position: absolute;
+		background-color: var(--oex-off-white);
+		z-index: 1;
+		min-width: 100%;
+		box-shadow: 2px 0px 16px rgba(207, 207, 207, 0.1),
+			-2px 0px 4px rgba(207, 207, 207, 0.1),
+			0px 2px 12px rgba(207, 207, 207, 0.1),
+			0px -2px 16px rgba(207, 207, 207, 0.1);
+
+		& label {
+			display: flex;
+			gap: 3px;
+			align-items: flex-start;
+			width: 100%;
+			padding: 5px 3px;
+			cursor: pointer;
+
+			&:hover {
+				background-color: #ffe8da;
+			}
+		}
+
+		& input:checked {
+			color: var(--oex-orange);
+			background-color: var(--oex-orange);
+		}
+
+		& input[type='checkbox']:checked {
+			background-color: var(--oex-orange);
+		}
+	}
+
+	@media ${({ theme }) => theme.breakpoints.above.md} {
+		border-left: 1px solid var(--text-colour-grey);
+	}
+`;
+
+const DivSection = styled.div`
+	display: flex;
+	align-items: center;
+	gap: 10px;
+	justify-content: center;
+	color: var(--text-colour-grey);
+	text-align: center;
+
+	&:hover {
+		color: var(--oex-orange);
+	}
+`;
+
+const DatePickerWrapper = styled.div`
```

Comment:
This is the same way I expected you'd style the radix modal instead of creating a CSS module

Reply:
I tried it but the couldn't get it correctly. It has specific style inheritance. The CSS module was quite easier to target the inherited styles. That's why. i will still use the radix again in the future for a feature, i'll try again to figure it out, then i can re-implement this.